modules/mobile/config/routes.rb

```diff
@@ -42,6 +42,7 @@
     post '/user/phones', to: 'phones#create'
     put '/user/phones', to: 'phones#update'
     delete '/user/phones', to: 'phones#destroy'
+    get '/community_care/eligibility/:service_type', to: 'cc_eligibility#show'
```

Comment:
Maybe I should rename 'specialty' to 'service type' in my PR. That's another place we should standardize naming.

Reply:
What do we think about the information architecture of `/community_care` being a top level resource? Could we nest it under appointments? Or if it's specific to requests, maybe even further down?

Reply:
It would get used for both direct and request scheduling. I'm on the fence on how/if we should group the routes. I like the idea putting this one under `appointments/community_care/eligibility/:service_type` as community cares eligibility will always related to appointments. But I'd still want to keep this as its own controller rather than make two separate custom actions on the appointment and appointment request controllers.